.gitignore

```diff
@@ -0,0 +1,6 @@
+/tests/__pycache__/
+/tuna/__pycache__/
+/tuna/dbBase/__pycache__
+/tuna/utils/__pycache__
```

Comment:
it seems like u added __pycache__ elements to your push. Please remove any __pycache__ folders